recipes/pastix/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "pastix" %}
+{% set version = "6.4.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url: https://files.inria.fr/pastix/releases/v6/pastix-{{ version }}.tar.gz
+  sha256: 891d426188eed56c1075fb34d2d80132593a1536ffc05cf333567f68a4811e55
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('pastix', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ stdlib('c') }}                 # [unix]
+    - {{ compiler('c') }}               # [unix]
+    - {{ compiler('cxx') }}             # [unix]
+    - {{ compiler('fortran') }}         # [unix]
+    - {{ compiler('m2w64_c') }}         # [win]
+    - {{ compiler('m2w64_cxx') }}       # [win]
+    - {{ compiler('m2w64_fortran') }}   # [win]
+    - cmake
+    - ninja
+    - pkgconfig
+
+  host:
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
+    # - libscotch # no 64bit support
+    - libhwloc
+    - metis
+
+
+test:
+  commands:
+    # no idea how to test pastix...
+
+about:
+  home: https://solverstack.gitlabpages.inria.fr/pastix/
+  summary: 'library for solving sparse real and complex linear systems of equations'
+  license: LGPL-3.0-or-later
+  license_file: LICENSE
+  description: |
+    PaStiX (Parallel Sparse matriX package) is a scientific library that 
+    provides a high performance parallel solver for very large sparse linear 
+    systems based on direct methods. Numerical algorithms are implemented 
+    in single or double precision (real or complex) using LLt, LDLt and LU 
+    with static pivoting (for non symmetric matrices having a symmetric pattern). 
+    This solver also provides some low-rank compression methods to reduce the 
+    memory footprint and/or the time-to-solution.
+  doc_url: https://solverstack.gitlabpages.inria.fr/pastix/
```

Ah, nevermind, just found it. Should be added here so that these sort of investigations can be done easily
```suggestion
  doc_url: https://solverstack.gitlabpages.inria.fr/pastix/
  dev_url: https://gitlab.inria.fr/solverstack/pastix
```